content/en/blog/2020/on-demand-egress-gateways/index.md

```diff
@@ -0,0 +1,500 @@
+---
+title: On Demand Egress Gateways
+subtitle: Deploy Egress Gateways independently in a seamless way
+description: Deploying Egress Gateways independently to have fine-grained control of egress communication.
+publishdate: 2020-11-07
+attribution: Antonio Berben (At Deutsche Telekom - PAN-NET)
+keywords: [configuration,egress,gateway,external,service]
+target_release: 1.8.0
+---
+
+At [`Deutsche Telekom::PAN-NET`](https://pan-net.cloud/aboutus), we have embraced `Istio` as the umbrella to cover our services.
+
+Unfortunately, there are services which have not been migrated to `kubernetes` yet or, simply, cannot be. Therefore, those ones have to remain as external services.
+
```

Comment:
```suggestion

Unfortunately, there are services which have not yet been migrated to Kubernetes, or cannot be. 

```

removing part about external services as it makes it seem like they're a named thing called an ExternalService which we haven't defined yet